### DISCUSSION TITLE
Only the statistics values are exposed

### DISCUSSION THREAD

**Discussion by erraa** [phaag/nfdump_influx#3] [Q&A] [answered]
First of all thank you for writing this tool. Secondly I am interested to know why we are not sending all data to influx such as srcaddr, dstaddr or intf?

**Comment by phaag** [comment, marked as answer]
nfinflux is still experimental. I am not sure, if influxdb would scale up to 30-50k flows/s if you push all flow details into the DB.
The idea is to use the influx DB for statistic and graph data source either with Grafana or potentially a new incarnation of NfSen.

Additional stats such as top10 or others may go into influx DB. So far I am not sure if it is a good idea to pump all records into the DB.
